src/imitation/policies/trainer.py

```diff
@@ -0,0 +1,83 @@
+"""Training policies with a specifiable reward function and collect trajectories."""
+from typing import List, Union
+
+from stable_baselines3.common.base_class import BaseAlgorithm
```

Comment:
Google Python style guide: import modules not classes. (mentioned previously in other files)